build_docs

```diff
@@ -231,13 +231,7 @@ def run_build_docs(args):
                         '%s:%s:ro' % (auth_sock_dir, auth_sock_dir)
                 ])
                 docker_args.extend(['-e', 'SSH_AUTH_SOCK=%s' % auth_sock])
-            known_hosts = realpath('%s/.ssh/known_hosts' % environ['HOME'])
-            if exists(known_hosts):
-                # If we have known_hosts mount them into the container so it
-                # won't ask about github
-                docker_args.extend([
-                        '-v',
-                        '%s:/tmp/.ssh/known_hosts:ro,cached' % known_hosts])
+            docker_args.extend(['-v', '%s/github.known_hosts:/tmp/.ssh/known_hosts:ro,cached' % DIR])
```

Comment:
This line is failing the lint due to being too long. Are you able to shorten it or move it to multiple lines?

```
09:15:07 build_docs:234:80: E501 line too long (101 > 79 characters)
```

Reply:
Oops. Fixed in commit c8baf7d2. Thanks.